.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -0,0 +1,46 @@
+name: 🐛 Report a bug
+description: '___'
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for reporting a bug! ⛰
+  - type: checkboxes
+    id: terms
```

Comment:
Are these IDs needed? If not, let's drop them

Reply:
According to https://gh-community.github.io/issue-template-feedback/structured/,
no, `id` is optional.
So, I just removed all 6 `id`.